.gitmodules

```diff
@@ -106,3 +106,7 @@
 [submodule "vendor/nim-web3"]
 	path = vendor/nim-web3
 	url = https://github.com/status-im/nim-web3.git
+[submodule "vendor/rln"]
+	path = vendor/rln
+	url = https://github.com/kilic/rln
+	branch = full-node
```

Comment:
Curious, what is this branch for? Vs master

Reply:
@oskarth Good question! this is a feature branch that most of the things we need are under this branch, as far as I checked with Onur, full-node branch will be ultimately merged with the master